<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-MSUCRPV" name="Test" tests="4" failures="1" timestamp="26 Jun 2019 03:31:59 GMT" time="37.065" errors="0">
  <testcase name="test4" time="4.843" classname="pages.DemoTestng">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at pages.DemoTestng.test4(DemoTestng.java:99)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase name="test3" time="14.537" classname="pages.DemoTestng"/>
  <testcase name="test1" time="7.436" classname="pages.DemoTestng"/>
  <testcase name="test2" time="6.216" classname="pages.DemoTestng"/>
</testsuite> <!-- Test -->
